### PR TITLE
Aruha 953 remove read write scopes

### PR DESCRIPTION
**Pull request by lmontrieux**
[ARUHA-953: Remove read_scopes and write_scopes](https://techjira.zalando.net/browse/ARUHA-953)

## Description
This PR removes the code for read_scopes and write_scopes, as well as the feature toggle for their enforcement on the consumption and production paths, respectively.

## Review
- [x] Tests
- [x] Documentation
- [x] CHANGELOG

## Deployment Notes
Deployment of this PR should not be done until NO event type has defined read_scopes or write_scopes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Removed
+
```
Comment (v-stepanov) [inline CHANGELOG.md]: please remove empty line

File: src/main/java/org/zalando/nakadi/domain/EventTypeBase.java [hunk 2]
```diff
@@ -201,22 +189,6 @@ public void setOptions(final EventTypeOptions options) {
         this.options = options;
     }
 
-    public Set<String> getWriteScopes() {
-        return Collections.unmodifiableSet(writeScopes);
-    }
-
-    public void setWriteScopes(final Set<String> writeScopes) {
```
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/domain/EventTypeBase.java]: what will happen if somebody will try to create ET having this property? Will request fail or will Nakadi just ignore this property?

Comment (lmontrieux) [inline src/main/java/org/zalando/nakadi/domain/EventTypeBase.java]: I tested that locally, Nakadi will ignore the property:
```
curl -v -XPOST -H "Content-Type: application/json" http://localhost:8080/event-types -d '{
  "name": "order_received2",
  "owning_application": "acme-order-service",
  "category": "business",
  "partition_strategy": "hash",
  "partition_key_fields": ["order_number"],
  "enrichment_strategies": ["metadata_enrichment"],
  "default_statistic": {
    "messages_per_minute": 1000,
    "message_size":5,
    "read_parallelism":1,
    "write_parallelism": 1
  },
  "schema": {
    "type": "json_schema",
    "schema": "{ \"properties\": { \"order_number\": { \"type\": \"string\" } } }"
  }, "read_scopes":["my.test.scope"], "write_scopes":["my.test.scope","my.test.scope2"]
}'
Note: Unnecessary use of -X or --request, POST is already inferred.
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8080 (#0)
> POST /event-types HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.54.0
> Accept: */*
> Content-Type: application/json
> Content-Length: 582
>
* upload completely sent off: 582 out of 582 bytes
< HTTP/1.1 201
< X-Flow-Id: geIHbEVPR3dIuzBeFCwKvVLR
< X-Content-Type-Options: nosniff
< X-XSS-Protection: 1; mode=block
< Cache-Control: no-cache, no-store, max-age=0, must-revalidate
< Pragma: no-cache
< Expires: 0
< X-Frame-Options: DENY
< X-Application-Context: application:8080
< Content-Length: 0
< Date: Tue, 01 Aug 2017 13:16:49 GMT
<
* Connection #0 to host localhost left intact
```

File: CHANGELOG.md (hunk 1 above)
Comment (lmontrieux) [inline CHANGELOG.md]: @v-stepanov f69056c

**Comment by v-stepanov** [comment]
👍

**Comment by lmontrieux** [comment]
@v-stepanov there were merge conflicts, would you mind re-approving?

**Comment by v-stepanov** [comment]
👍

**Comment by v-stepanov** [comment]
I approved but there is a checkstyle issue

**Comment by lmontrieux** [comment]
@v-stepanov checkstyle issue fixed in 37fdf4f

**Comment by v-stepanov** [comment]
👍

**Comment by lmontrieux** [comment]
👍

**Comment by v-stepanov** [comment]
👍

**Comment by lmontrieux** [comment]
👍

**Pull request merged**